lib/iris/tests/test_cdm.py

```diff
@@ -1199,7 +1199,8 @@ def test_save_and_merge(self):
             self.assertEqual(len(merged_cubes), 1, "expected a single merged cube")
             merged_cube = merged_cubes[0]
             self.assertEqual(merged_cube.dtype, dtype)
-            self.assertEqual(merged_cube.data.fill_value, fill_value)
+            # Note: original data fill-value is *ignored*.
```

Comment:
Sorry, it should probably say "check that original data fill-value is *ignored*".
I do think that is worth pointing out, though,